src/bluesearch/entrypoint/database/topic_extract.py

```diff
@@ -168,6 +173,33 @@ def run(
                     },
                 }
             )
+    elif source == "pubmed":
+        for path in inputs:
+            logger.info(f"Processing {path}")
+            articles = ElementTree.parse(str(input_path))
```

Comment:
```suggestion
            articles = ElementTree.parse(input_path)
```
I tried w/o `str` casting and it seemed to work, is it necessary here?